### DISCUSSION TITLE
Stencil dragEndClone accpect async function

### DISCUSSION THREAD

**Discussion by zta7** [clientIO/joint#2083] [Q&A] [answered]
### Introduction

I want to open a Dialog for user to input a 'name' value, but it's seems impossible right now for Stencil dragEndClone function.
Something like this~ Will u support it or i should make it work without use Stencil library~ 
![image](https://user-images.githubusercontent.com/30122066/226569500-cd0dc7eb-b62b-4928-8959-35eee609fc2f.png)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
The `dragEndClone()` option callback must stay synchronous.

But you can use the Stencil API and events to implement this. Here's a similar [example](https://codepen.io/jointjs/pen/zYpVdoV).
